foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/model/OpenAiEmbeddingParameters.java

```diff
@@ -40,6 +40,21 @@ public class OpenAiEmbeddingParameters {
   @Setter(onParam_ = @Nullable)
   private String inputType;
 
+  /**
+   * The format to return the embeddings in. Can be either `float` or `base64`. Defaults to `float`.
+   */
+  @JsonProperty("encoding_format")
+  @Setter(onParam_ = @Nullable)
+  private String encodingFormat;
+
+  /**
+   * The number of dimensions the resulting output embeddings should have. Only supported in
+   * `text-embedding-3` and later models.
+   */
+  @JsonProperty("dimensions")
+  @Setter(onParam_ = @Nullable)
+  private Integer dimensions;
```

Comment:
(Major)

> The number of dimensions the resulting output embeddings should have.

The only text embedding model available for testing is `TEXT_EMBEDDING_ADA_002`.
Unfortunately it does not change output upon setting this field. No matter `null`, `0`, `1`, or `5`.

> Only supported in `text-embedding-3` and later models.

Should we release something that we cannot test?